Config.cs

```diff
@@ -1,32 +1,56 @@
-﻿using CounterStrikeSharp.API.Core;
+using CounterStrikeSharp.API.Core;
 using System.Text.Json.Serialization;
 
 namespace WeaponPaints
 {
+    public class Messages
+    {
+        [JsonPropertyName("WebsiteMessageCommand")]
+        public string WebsiteMessageCommand { get; set; } = "Visit {WEBSITE} where you can change skins.";
+        [JsonPropertyName("SynchronizeMessageCommand")]
+        public string SynchronizeMessageCommand { get; set; } = "Type !wp to synchronize chosen skins.";
+        [JsonPropertyName("KnifeMessageCommand")]
+        public string KnifeMessageCommand { get; set; } = "Type !knife to open knife menu.";
+        [JsonPropertyName("CooldownRefreshCommand")]
+        public string CooldownRefreshCommand { get; set; } = "You can't refresh weapon paints right now.";
+        [JsonPropertyName("SuccessRefreshCommand")]
+        public string SuccessRefreshCommand { get; set; } = "Refreshing weapon paints.";
+        [JsonPropertyName("ChosenKnifeMenu")]
+        public string ChosenKnifeMenu { get; set; } = "You have chosen {KNIFE} as your knife.";
+        [JsonPropertyName("KnifeMenuTitle")]
+        public string KnifeMenuTitle { get; set; } = "Knife Menu.";
+    }
+
     public class WeaponPaintsConfig : BasePluginConfig
     {
-        public override int Version { get; set; } = 1;
+        public override int Version { get; set; } = 2;
 
         [JsonPropertyName("DatabaseHost")]
-        public string DatabaseHost { get; set; } = "";
+        public string DatabaseHost { get; set; } = "localhost";
 
         [JsonPropertyName("DatabasePort")]
         public int DatabasePort { get; set; } = 3306;
 
         [JsonPropertyName("DatabaseUser")]
-        public string DatabaseUser { get; set; } = "";
+        public string DatabaseUser { get; set; } = "DBUSER";
 
         [JsonPropertyName("DatabasePassword")]
-        public string DatabasePassword { get; set; } = "";
+        public string DatabasePassword { get; set; } = "DBPASSWORD";
 
         [JsonPropertyName("DatabaseName")]
-        public string DatabaseName { get; set; } = "";
+        public string DatabaseName { get; set; } = "DBDATABASE";
```

Comment:
please, make it empty so we can throw error, if user does not insert credentials